introduction.md

```diff
@@ -0,0 +1,18 @@
+Ruby Object Mapper (ROM) is a Ruby library with the goal to provide powerful object mapping capabilities without limiting the full power of your datastore. ROM is based on several concepts and decisions that differentiate it from your normal Ruby ORM:
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.